.circleci/deploy_to_kubernetes

```diff
@@ -1,25 +1,30 @@
-#!/bin/sh -eu
+#!/bin/sh -e
```

Comment:
This was referring to line 9, which is now `printf "usage: deploy_to_kubernetes namespace\n"` instead of `echo -e`. I'll update the commit and notes, thanks!